src/resource/page/ResourcePage.test.tsx

```diff
@@ -27,6 +28,21 @@ const testResource: Resource = {
   },
 };
 
+const testDatePeriod: DatePeriod = {
+  id: '1',
```

Comment:
Small thing: If I remember correct, the id is also type number?

Reply:
I'll fix it